public/css/main.css

```diff
@@ -66,6 +66,7 @@
 }
 .copy-button-group .copy-button {
     padding-left: 4px;
+    padding-right: 4px;
```

Comment:
Could this just be `padding: 0 4px;`?

Reply:
By doing that it will remove the top/bottom padding that the dropdown items have right now. If you are fine with that I will change it.

Reply:
Ah yeah, selector priority issues. This will be fine then.